src/vm/appdomain.cpp

```diff
@@ -3020,7 +3012,7 @@ void SystemDomain::SetupDefaultDomain()
 
         {
             GCX_PREEMP();
-            InitializeDefaultDomain(TRUE);
+            InitializeDefaultDomain();
```

Comment:
Is this call basically only logging/tracing/test hooks - I don't see any actual functionality in it.
I assume that is because the default domain setup is done during startup elsewhere, so here the domain is already setup so there's no need to do anything.
It's just that the Initialize doesn't seem to actually initialize anything.
Also - do we need the debugger magic around this call? I don't think it would run any actual managed code anyway... right?
Also - while you're here, can you please fix the typos on the comment for the debugger (if we keep it) ?

Reply:
I'm also trying to figure out where is this actually called from. I tracked it down to InitializeEE / EnsureEEStarted, but that only seems to be called from COM entry points. So probably not used at all right now...
I'm not saying to clean all of that up, I'm just trying to understand what's the scenario here, and why it's OK to not initialize the default domain (And if we can maybe assert if it didn't happen...).

Reply:
Yes, none of this was doing anything useful after taking closer look - deleted.

The regular hosting goes through these COM entrypoint. E.g. the stack typically looks like this: 

```
CoreCLR!EEStartup [c:\coreclr\src\vm\ceemain.cpp @ 1135] 
CoreCLR!EnsureEEStarted+0x1a9 [c:\coreclr\src\vm\ceemain.cpp @ 354] 
CoreCLR!CorRuntimeHostBase::Start+0xd6 [c:\coreclr\src\vm\corhost.cpp @ 219] 
CoreCLR!CorHost2::Start+0x148 [c:\coreclr\src\vm\corhost.cpp @ 177] 
```

Reply:
Thanks for the explanation